itou/templates/includes/deactivate_member.html

```diff
@@ -2,21 +2,30 @@
 
 <div class="content-small">
     <h1>Retrait d'un collaborateur</h1>
-    <div class="mt-4 alert alert-warning">
-        Vous allez retirer <b>{{ target_member.get_full_name }}</b>
-        de votre organisation <b>({{ structure.display_name }})</b>.
-    </div>
+    <p>
+        Êtes-vous sûr de vouloir retirer <b>{{ target_member.get_full_name }}</b> de votre organisation <b>({{ structure.display_name }})</b> ?
+    </p>
 
+    <p class="mb-0">Une fois retiré :</p>
     <ul>
-        <li>Le tableau de bord de votre structure ne lui sera plus accessible.</li>
-        <li>Les modifications et ajouts réalisés sont toujours accessibles par les autres membres de votre organisation.</li>
         <li>Cet utilisateur sera notifié par e-mail de son retrait de votre organisation.</li>
-        <li>En cas d'erreur, vous pourrez toujours lui renvoyer une invitation pour rejoindre votre organisation.</li>
-        {% if current_user.is_employer or current_user.is_prescriber %}
+        <li>Le tableau de bord de votre structure ne lui sera plus accessible.</li>
+        {% if request.user.is_prescriber %}
+            <li>Les candidatures effectuées par ce collaborateur resteront dans le tableau de bord de l'organisation</li>
```

Comment:
```suggestion
            <li>Les candidatures effectuées par ce collaborateur resteront dans le tableau de bord de l'organisation.</li>
```

Reply:
```suggestion
            <li>Les candidatures effectuées par ce collaborateur restent dans le tableau de bord de l'organisation.</li>
```
Pour garder le même temps qu’en dessous.

Reply:
J'ai tout passé au future plutot